build.rs

```diff
@@ -0,0 +1,14 @@
+#[cfg(not(any(target_os = "macos", windows)))]
+extern crate pkg_config;
+
+#[cfg(not(any(target_os = "macos", windows)))]
+fn main() {
+    if pkg_config::Config::new().atleast_version("20.0.14").probe("freetype2").is_ok() {
```

Comment:
```suggestion
fn main() {
    #[cfg(not(any(target_os = "macos", windows)))]
    if pkg_config::Config::new().atleast_version("20.0.14").probe("freetype2").is_ok() {
```

I think this would be much easier, with the new annotations support?